### PR TITLE
[bitnami/external-dns] feat: add revisionHistoryLimit

### PR DESCRIPTION
**Pull request by KyriosGN0**
### Description of the change

Added revisionHistoryLimit parameter for external-dns

### Benefits

Users will be able to set this parameters to control the number of replicasets to keep in k8s

### Possible drawbacks

None

### Applicable issues

None

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by KyriosGN0** [comment]
hey, @jotamartos could you perhaps take a look at mr PR ? 
is there something missing ?

File: bitnami/external-dns/Chart.yaml [hunk 1]
```diff
@@ -28,4 +28,4 @@ maintainers:
 name: external-dns
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/external-dns
-version: 7.0.0
+version: 7.0.1
```
Comment (jotamartos) [inline bitnami/external-dns/Chart.yaml:31]:
```suggestion
version: 7.1.0
```

File: bitnami/external-dns/values.yaml [hunk 2]
```diff
@@ -88,6 +88,9 @@ image:
   ##   - myRegistryKeySecretName
   ##
   pullSecrets: []
+## @param revisionHistoryLimit sets number of replicaset to keep in k8s
+##
+revisionHistoryLimit: 3
```
Comment (jotamartos) [inline bitnami/external-dns/values.yaml:93]:
```suggestion
revisionHistoryLimit: 10
```

The default value is 10

https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#clean-up-policy

**Review by jotamartos — CHANGES_REQUESTED** [review]
Thanks for creating the PR. Could you please take a look at my suggestions? Thanks

File: bitnami/external-dns/values.yaml (hunk 2 above)
Comment (KyriosGN0) [reply bitnami/external-dns/values.yaml]: fixed

File: bitnami/external-dns/Chart.yaml (hunk 1 above)
Comment (KyriosGN0) [reply bitnami/external-dns/Chart.yaml]: fixed

**Comment by KyriosGN0** [comment]
hey @jotamartos fixed your comments
thanks for the review!

**Review by jotamartos — APPROVED** [review]

**Review by jotamartos — APPROVED** [review]

**Pull request merged**